tests/ImportsNotSeparated/__snapshots__/ppsi.spec.ts.snap

```diff
@@ -96,10 +99,11 @@ function add(a:number,b:number) {
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 // I am top level comment in this file.
 // I am second line of top level comment in this file.
+
 import "./commands";
 import React from "react";
-
 // Comment
+
```

Comment:
🤨 where did this newline come from?

Reply:
A newline appears when dealing with a “bottom-of-imports” comment.

Your request to change the logic so that nothing attaches (to top-level-decls) as trailing except for same-line comments address this a bit more cleanly?

Reply:
Please check out this case again (after I push up my latest changes)